src/protocolsupport/protocol/pipeline/version/v_pe/PEPacketEncoder.java

```diff
@@ -9,66 +9,7 @@
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.Explosion;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.HeldSlot;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.InventoryClose;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.InventoryConfirmTransaction;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.InventoryData;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.InventoryOpen;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.InventorySetItems;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.InventorySetSlot;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.KeepAlive;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.KickDisconnect;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.Map;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.PlayerAbilities;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.PlayerListSetEntry;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.ServerDifficulty;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SetExperience;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SetHealth;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SetPosition;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SpawnExpOrb;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SpawnGlobal;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SpawnLiving;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SpawnNamed;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SpawnObject;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SpawnPainting;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.SpawnPosition;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.StartGame;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.TimeUpdate;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.Title;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.UnloadChunk;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.VehiclePassengers;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.WorldCustomSound;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.WorldEvent;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.WorldParticle;
-import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.WorldSound;
+import protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe.*;
```

Comment:
intellij is evil like this >.< it sucks, but should try to keep imports explicit